internal/builder/builder.go

```diff
@@ -148,11 +148,45 @@ func calculateMTKVariable(name string, build Builder, vars []variables.LagoonEnv
 
 // imagePatternParser parses the image pattern
 func imagePatternParser(pattern string, build Builder) string {
-	pattern = strings.Replace(pattern, "${database}", build.MTK.Database, 1)
+	var clean_database string
+	clean_database = replaceDoubleSpecial(build.MTK.Database)
+	pattern = strings.Replace(pattern, "${database}", clean_database, 1)
```

Comment:
Style change, variables to be camelCase
```suggestion
	cleanDatabase := replaceDoubleSpecial(build.MTK.Database)
	pattern = strings.Replace(pattern, "${database}", cleanDatabase, 1)
```